### ISSUE TITLE
Website isn't accessible - colour contrast

### ISSUE THREAD

**Issue by MarkitkaM** [IsThereAnyDeal/IsThereAnyDeal-Issues#2757]
**Steps to reproduce:**
1. Go to https://new.isthereanydeal.com/
2. Check the accessibility of the website colour contrast https://webaim.org/resources/contrastchecker/

**Actual Result:** Test of accessibility of the website colour contrast FAILED.
![image](https://user-images.githubusercontent.com/88401908/158959369-4538d04e-eafe-4f59-91da-38df05fc4ec0.png)

**Expected result:** Test of accessibility of the website colour contrast PASSED.

**Comment by MarkitkaM** [comment]
Passed
![image](https://user-images.githubusercontent.com/88401908/158959615-97ecf19d-5b02-483b-8c14-850fe8390b06.png)
